<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <widget class="QLineEdit" name="signup_pw2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="signup_pw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PW</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signup_id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PW확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signup_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="idcheck_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>중복확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
